test/OpenTelemetry.Exporter.OneCollector.Benchmarks/OpenTelemetry.Exporter.OneCollector.Benchmarks.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <Description>Benchmark project for OpenTelemetry .NET OneCollectorExporter.</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(SupportedNetTargets)</TargetFrameworks>
```

Comment:
It will be hard to bring it back.
Maybe we could create new one SupportedNetTargetsWithoutNet6. Future renaming will be easier.

Reply:
Please check https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2117#discussion_r1781506008.

Reply:
But for sure, I can introduce a new prop and update packages where I already removed .NET 6.

Reply:
I added a new SupportedNetTargetsWithoutNet6 that should be renamed later.